user/i2pd/files/i2pd

```diff
@@ -0,0 +1,9 @@
+type = process
+command = /usr/bin/i2pd --service --certsdir /usr/share/i2pd/certificates --conf /usr/share/i2pd/i2pd.conf --tunconf /usr/share/i2pd/tunnels.conf
```

Comment:
it's not correct to force config files to go to `/usr`

`/usr` should merely contain a default (if such default is needed at all) that the administrator can override with a file in `/etc`

i.e. it shouldn't be necessary for an `/etc` file to exist, but all user configuration should be done there

Reply:
same applies to the others btw